sdk/core/ci.yml

```diff
@@ -31,29 +31,38 @@ pr:
     - eng/mgmt/
     - eng/common/
 
+parameters:
+  - name: Artifacts
+    type: object
+    default:
+      - name: Azure.Core
+        safeName: AzureCore
+      - name: Azure.Core.Experimental
+        safeName: AzureCoreExperimental
+      - name: Azure.Core.Expressions.DataFactory
+        safeName: AzureCoreExpressionsDataFactory
+      - name: Microsoft.Azure.Core.NewtonsoftJson
+        safeName: MicrosoftAzureCoreNewtonsoftJson
+      - name: Microsoft.Azure.Core.Spatial
+        safeName: MicrosoftAzureCoreSpatial
+      - name: Microsoft.Azure.Core.Spatial.NewtonsoftJson
+        safeName: MicrosoftAzureCoreSpatialNewtonsoftJson
+        # Currently due to size constraints with 1es pipeline templates we cannot include ALL release stages in one build
```

Comment:
This feels like it is going a little too far. We should be able to release these in one pipeline, but if they don't regularly ship these together then we could split them into a new pipeline. However I also think it is fine to block on this until we get the size limit increased.